packages/sitecore-jss-nextjs/src/index.ts

```diff
@@ -139,6 +139,8 @@ export {
   EditFrame,
   FEaaSComponent,
   FEaaSComponentProps,
+  getFEAASLibraryStylesheetURLs,
+  FEAAS_CDN,
```

Comment:
Doesn't look like we need to export FEAAS_CDN (and I don't think we'd want to even - should be internal).